package.json

```diff
@@ -84,18 +85,21 @@
     "react-live-clock": "^4.0.5",
     "react-modal": "^3.11.2",
     "react-redux": "^7.1.3",
+    "react-scripts": "3.4.1",
+    "react-stopwatch": "^2.0.4",
     "reactstrap": "^8.4.1",
     "redux": "^4.0.4",
     "redux-persist": "^6.0.0",
     "resolve": "1.12.0",
     "resolve-url-loader": "3.1.0",
     "sass-loader": "7.2.0",
     "sdp-transform": "^2.14.0",
-    "semver": "6.3.0",
+    "semver": "7.3.2",
+    "serialize-javascript": "^5.0.1",
```

Comment:
?

Reply:
To appease the GitHub security warning gods.